Movie/Movie/Presentation/DetailsMovie/Presenter/DetailsMoviePresenter.swift

```diff
@@ -0,0 +1,75 @@
+// DetailsMoviePresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Презентер экрана деталей о фильме
+final class DetailsMoviePresenter: DetailsMoviePresenterProtocol {
+    // MARK: - Public Properties
+
+    let networkService: NetworkServiceProtocol
+
+    weak var view: DetailsMovieViewProtocol?
+    var router: MoviesRouterProtocol?
+    var movieDetails: MovieDetails?
+    var movieId: Int?
+    var recommendationMovies: [RecommendationMovie] = []
+
+    // MARK: - Initializers
+
+    required init(
+        view: DetailsMovieViewProtocol,
+        networkService: NetworkServiceProtocol,
+        movieId: Int,
+        router: MoviesRouterProtocol
+    ) {
+        self.view = view
+        self.networkService = networkService
+        self.movieId = movieId
+        self.router = router
+        fetchRecommendationMovies()
+        fetchMovieDetails()
+    }
+
+    // MARK: - Public Methods
+
+    func fetchMovieDetails() {
+        guard let movieId else { return }
+        networkService.fetchMovie(id: movieId) { [weak self] result in
+            guard let self else { return }
+            switch result {
+            case .failure:
+                DispatchQueue.main.async {
```

Comment:
```suggestion
                DispatchQueue.main.async {
```
ЛУчше 1 раз внутри метода сделай чем тут в каждом кейсе